system/ee/ExpressionEngine/Addons/pages/tab.pages.php

```diff
@@ -294,6 +333,8 @@ public function prepareSitePagesData($entry, $values)
 
                 $page = '/' . trim($page, '/');
 
+                //if we got here, and page URI is not unique, generate new one
+
```

Comment:
@intoeetive I see this comment added... should there be additional code here?

Reply:
Removed this comment, it's not relevant anymore - the necessary code is in `clone` method